README.rst

```diff
@@ -176,12 +176,16 @@ init(autoreset=False):
 init(strip=None):
     Pass ``True`` or ``False`` to override whether ansi codes should be
     stripped from the output. The default behaviour is to strip if on Windows
-    or if output is redirected (not a tty).
+    or if output is redirected (not a tty), and if the ``FORCE_COLOR``
+    environment variable is not set (with any value) or ``NO_COLOR`` is
```

Comment:
IMHO there are a little too many "and"s and "or"s in this sentence and it would be good to break it down for clarity.

Reply:
I could rewrite the comment as a series of bullet-points

Reply:
Ok - I rewrote it as a set of bullet-points. It seems slightly easier to read now.